docs/core/integration-quality-scale/rules/iqs028.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "IQS028 - If making HTTP requests, support passing in websession"
+---
+
+IQS028 is as follows:
+
+If making HTTP requests, support passing in websession
+
+## Reasoning
+
+Since a lot of devices and services are connected to via HTTP, the amount of websessions that are active can be a lot.
+To improve the efficiency of those websessions, it is recommended to support passing in a websession.
+These sessions are tweaked towards being used in Home Assistant and is more efficient.
```

Comment:
```suggestion
Since many devices and services are connected via HTTP, the number of active web sessions can be high.
To improve the efficiency of those web sessions, it is recommended to support passing in a web session.
These sessions are tweaked towards being used in Home Assistant and is more efficient.
```